plugins/net_plugin/net_plugin.cpp

```diff
@@ -3019,6 +3020,7 @@ namespace eosio {
       fc::raw::unpack( peek_ds, bh );
       const block_id_type blk_id = bh.calculate_id();
       const uint32_t blk_num = last_received_block_num = block_header::num_from_id(blk_id);
+      my_impl->last_block_num_received = blk_num;
```

Comment:
shouldn't it be something like:
`my_impl->last_block_num_received = std::max(blk_num, my_impl->last_block_num_received);`

or are we guaranteed to receive blocks from only one connection at a time?